spec/lib/resources_spec.rb

```diff
@@ -1,4 +1,4 @@
-# frozen_string_literal: true
+provider# frozen_string_literal: true
```

Comment:
Copy-paste issue? I think this is breaking the tests

Reply:
https://github.com/cerner/fhir.cerner.com/pull/430/commits/56870f0ff4573ad640d54226948788ec51129ab0#diff-0c4be351aacc6e5786906aa41d053668R1